api_app/api/routes/shared_services.py

```diff
@@ -93,6 +93,9 @@ async def patch_shared_service(shared_service_patch: ResourcePatch, response: Re
         raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=strings.ETAG_CONFLICT)
     except ValidationError as v:
         raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=v.message)
+    except (MajorVersionUpdateDenied, TargetTemplateVersionDoesNotExist, VersionDowngradeDenied) as e:
+        logging.error(str(e), exc_info=e)
```

Comment:
Is this a "real" error we as developers should investigate?
(same comment for all the routes)

Reply:
No, those are validation exceptions when the user tries to upgrade a major version or downgrade a version, or the target version is not registered.
Do you suggest to remove the logging?

Reply:
It doesn't look like you log extra information not returned in the http error... I'd remove it